src/mixins/regions.js

```diff
@@ -89,9 +89,9 @@ export default {
   },
 
   _buildRegionFromObject: function(definition) {
-    var RegionClass = definition.regionClass || this.regionClass;
+    const RegionClass = definition.regionClass || this.regionClass;
 
-    var options = _.omit(definition, 'regionClass');
+    let options = _.omit(definition, 'regionClass');
```

Comment:
I think `const` works here.  Since options is an object the "constant" is just the reference to the object.